src/app/components/BibPage/BibDetails.jsx

```diff
@@ -1,39 +1,251 @@
+                    <li key={i}>
+                      <Link
+                        onClick={e => this.newSearch(e, url)}
+                        title={`Make a new search for ${fieldLabel}: ${valueObj.prefLabel}`}
+                        to={`/search?${url}`}
+                      >
+                        {valueObj.prefLabel}
+                      </Link>
+                    </li>
+                  );
+                })
+              }
+            </ul>
+          ),
+        };
+        // NOTE: Right now this is not working because we removed the `linkable` property.
+        // We added this because not all fields should be linkable. For example, maybe we
+        // want `materialType` to be linkable in the UI but not `issuance`.
+      } else if (field.linkable) {
+        // When we want all the links to be in sentence form separated by a comma:
+        // e.g. Benjamin Franklin, Thomas Jefferson, George Washington
+        if (fieldValue === 'contributorLiteral') {
+          return {
+            term: fieldLabel,
+            definition: bibValues.map((value, i) => {
+              const comma = bibValues.length > 1 ? ', ' : ' ';
+              const url = `filters[${fieldValue}]=${value}`;
+              return (
+                <span key={i}>
+                  <Link
+                    onClick={e => this.newSearch(e, url)}
+                    title={`Make a new search for ${fieldLabel}: "${value}"`}
+                    to={`/search?${url}`}
+                  >
+                    {value}
+                  </Link>{comma}
+                </span>
+              );
+            }),
+          };
+        }
+
+        // NOTE: because of the note above, this is never reached.
+        return {
+          term: fieldLabel,
+          definition: (
+            <ul>
+              {
+                bibValues.map((value, i) => {
+                  const url = `filters[${fieldValue}]=${value}`;
+                  return (
+                    <li key={i}>
+                      <Link
+                        onClick={e => this.newSearch(e, url)}
+                        title={`Make a new search for ${fieldLabel}: "${value}"`}
+                        to={`/search?${url}`}
+                      >
+                        {value}
+                      </Link>
+                    </li>
+                  );
+                })
+              }
+            </ul>
+          ),
+        };
+      }
 
-    return data.map((item) => ([
-      (<dt dangerouslySetInnerHTML={this.createMarkup(item.term)}></dt>),
-      (<dd>{item.definition}</dd>),
-    ]));
+      // Simple data display. This gets rendered the most.
+      return {
+        term: fieldLabel,
+        definition: (
+          <ul>
+            {bibValues.map((value, i) => <li key={i}>{value}</li>)}
+          </ul>
+        ),
+      };
+    });
   }
 
-  createMarkup(markup) {
-    return { __html: markup };
+  newSearch(e, query) {
```

Comment:
Yes, definitely a different issue/ticket. I think instead of `newSearch` going into `utils`, the contents and logic of the function can be there instead, especially if it's reused elsewhere.